include/EVT/io/SPI.hpp

```diff
@@ -29,6 +29,17 @@ enum class Pin;
 
 class SPI {
 public:
+    /**
+     * Represents potential errors that may take place when using the SPI
```

Comment:
NIT: It doesn't just represent errors. It also represents the lack of errors. I'm not sure how this could be worded, but it could be slightly better.